opencti-platform/opencti-front/src/private/components/data/ingestionCsv/IngestionCsvCreation.tsx

```diff
@@ -0,0 +1,343 @@
+      authentication_type: values.authentication_type,
+      authentication_value: authenticationValue,
+      current_state_date: values.current_state_date,
+      user_id: userId,
+    };
+    commit({
+      variables: {
+        input,
+      },
+      updater: (store) => {
+        insertNode(
+          store,
+          'Pagination_ingestionCsvs',
+          paginationOptions,
+          'ingestionCsvAdd',
+        );
+      },
+      onCompleted: () => {
+        setSubmitting(false);
+        resetForm();
+      },
+    });
+  };
+
+  const queryRef = useQueryLoading<CsvMapperFieldSearchQuery>(csvMapperQuery);
+
+  return (
+    <Drawer
+      title={t_i18n('Create a CSV ingester')}
+      variant={DrawerVariant.createWithPanel}
+    >
+      {({ onClose }) => (
+        <Formik<IngestionCsvCreationForm>
+          initialValues={{
+            name: '',
+            description: '',
+            uri: '',
+            csv_mapper_id: '',
+            authentication_type: 'none',
+            authentication_value: '',
+            current_state_date: null,
+            user_id: '',
+            username: '',
+            password: '',
+            cert: '',
+            key: '',
+            ca: '',
+          }}
+          validationSchema={ingestionCsvCreationValidation}
+          onSubmit={onSubmit}
+          onReset={onClose}
+        >
+          {({ submitForm, handleReset, isSubmitting, values }) => (
+            <Form style={{ margin: '20px 0 20px 0' }}>
+              <Field
+                component={TextField}
+                variant="standard"
+                name="name"
+                label={t_i18n('Name')}
+                fullWidth={true}
+              />
+              <Field
+                component={TextField}
+                variant="standard"
+                name="description"
+                label={t_i18n('Description')}
+                fullWidth={true}
+                style={fieldSpacingContainerStyle}
+              />
+              <Field
+                component={DateTimePickerField}
+                name="current_state_date"
+                textFieldProps={{
+                  label: t_i18n('Import from date (empty = all Csv possible items)'),
+                  variant: 'standard',
+                  fullWidth: true,
+                }}
+              />
+              <Field
+                component={TextField}
+                variant="standard"
+                name="uri"
+                label={t_i18n('CSV URL')}
+                fullWidth={true}
+                style={fieldSpacingContainerStyle}
+              />
+              {
+                queryRef
+                && <React.Suspense fallback={<Loader variant={LoaderVariant.inElement} />}>
+                  <CsvMapperField
+                    name="csv_mapper_id"
+                    isOptionEqualToValue={(option: Option, value: string) => option.value === value}
```

Comment:
```suggestion
                    isOptionEqualToValue={(option: Option, { value }: Option) => option.value === value}
```
and change accordingly the prop type in CSVMapperField